src/interactive-config.coffee

```diff
@@ -40,7 +40,12 @@ interactiveConfig.prompt = (config = {}, callback) ->
     name: "apiaryApiKey"
     message: "Please enter Apiary API key or leave empty for anonymous reporter"
     default: config['custom']?['apiaryApiKey']
-    when: (answers) -> (answers['apiary'] == true && ! config['custom']?['apiaryApiKey']?)
+    when: (answers) ->
+      # it's not GA on the Apiary side yet
+      if process.env['APIARY_BETA']?
+        (answers['apiary'] == true && ! config['custom']?['apiaryApiKey']?)
```

Comment:
@kuba-kubula you mean that condition statement or usage of `if`? I agree, but it was there before this PR :)